source/CIS/cis-2.rst

```diff
@@ -38,6 +38,10 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 General types and serialization
 -------------------------------
 
+.. note::
+
+  Integers are encoded in big endian unless stated otherwise.
```

Comment:
Do we ever use anything in big endian as part of this standard? If so, then I think it is better to be explicit when that is, instead of having some default.

Reply:
I don't think we ever use big endian in any smart contracts. We sometimes use LEB, but otherwise little endian.

So this default is odd.

Reply:
In that case, you should write that little-endian is the default unless otherwise noted.

Reply:
So @EmilLa1 will you just add here that all integers are encoded as little endian unless stated otherwise.